amundsen_application/static/js/features/ColumnList/ColumnLineage/index.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import { connect } from 'react-redux';
+
+import { GlobalState } from 'ducks/rootReducer';
+import * as React from 'react';
+import { emptyLineage } from 'ducks/tableMetadata/reducer';
+
+import { getColumnLineageLink } from 'config/config-utils';
+import './styles.scss';
+import { Lineage, TableMetadata } from 'interfaces/TableMetadata';
+
+interface ComponentProps {
+  columnName: string;
+  tableKey: string;
+}
+
+interface StateFromProps {
+  columnLineage: Lineage;
+  tableData: TableMetadata;
+}
+
+type Props = ComponentProps & StateFromProps;
+
+const getLink = (table) =>
+  `/table_detail/${table.cluster}/${table.database}/${table.schema}/${table.name}` +
+  `?source=column_lineage`;
+
+export class ColumnLineageList extends React.Component<Props> {
+  renderLineageLinks(entity, index) {
+    if (index >= 5) {
+      return null;
+    }
+    return (
+      <div>
+        <a href={getLink(entity)} target="_blank" rel="noreferrer">
+          {entity.schema}.{entity.name}
+        </a>
+      </div>
+    );
+  }
+
+  render() {
+    const { columnName, columnLineage, tableData } = this.props;
+    const { downstream_entities, upstream_entities } = columnLineage;
+    if (!downstream_entities.length && !upstream_entities.length) {
+      return null;
+    }
+    const externalLink = getColumnLineageLink(tableData, columnName);
+    return (
+      <section className="column-lineage-wrapper">
```

Comment:
I would make this an `<article>` if we consider this could be a "widget" that could be placed somewhere else in the site. The section tag doesn't really have any utility as it was mean to support a feature never implemented in browsers.